dojson/contrib/to_marc21/model.py

```diff
@@ -9,7 +9,7 @@
 
 """To MARC 21 model definition."""
 
-from collections import MutableMapping, MutableSequence
+from collections.abc import MutableMapping, MutableSequence
```

Comment:
Why have you changed this?

Reply:
I changed the import to abc to prevent Deprecation warning in future python version, in the test script and also in the Marc21 module.